docs/objects.md

```diff
@@ -4,17 +4,15 @@
 
 - [Tengo Objects](#tengo-objects)
   - [Object Interface](#object-interface)
-  - [Callable Interface](#callable-interface)
-  - [Indexable Interface](#indexable-interface)
-  - [Index-Assignable Interface](#index-assignable-interface)
-  - [Iterable Interface](#iterable-interface)
+  - [Callable Objects](#callable-interface)
+  - [Iterable Objects](#iterable-interface)
```

Comment:
broken link: `#callable-interface` `#iterable-interface`

Reply:
also should those be under `Object Interface` (one level deeper)? Or, just make them part of `Object` interface explanation, rather than separate sections.

Reply:
I've made them part of the Object Interface because ultimately they still rely on the CanItereate and Iterator methods.